themes/default/content/docs/support/faq.md

```diff
@@ -34,9 +34,9 @@ $ pulumi stack init acme-corp/widget-server
 
 The [Pulumi Service](https://app.pulumi.com/) allows you to transfer stacks from your individual account to any organization account you belong to as an administrator.
 
-To transfer a stack from your individual account to an organization, navigate to the project containing the stack you wish to migrate, then select the stack in question.
+To transfer a stack from your individual account to an organization, navigate to the Stacks page and select Transfer Stacks.
```

Comment:
i think our patten is to bold `Transfer stacks` also id drop the uppercase s to lowercase to align with the ui

Reply:
i saw that in the other pages, but the markdown looked totally different on this faq one. should i bold it anyway?

Reply:
yah you should be able to, its a .md file, surrounding it with `**` should do the trick